event-handlers/src/main/java/no/sikt/nva/nvi/events/db/DynamoDbEventToQueueHandler.java

```diff
@@ -116,9 +108,13 @@ private static String writeAsJsonString(DynamoDbChangeMessage updateMessage) {
   }
 
   private void splitIntoBatchesAndSend(DynamodbEvent input) {
-    splitIntoBatches(input.getRecords())
-        .map(DynamoDbEventToQueueHandler::mapToUpdateMessages)
-        .forEach(this::sendBatch);
+    var messages =
+        input.getRecords().stream()
+            .map(DynamoDbEventToQueueHandler::mapToUpdateMessage)
```

Comment:
Just a thought, we expect distinct, right?

Reply:
Yes, but I don't think it matters much (a duplicate message should just be extra load). I can add `distinct` here just in case though.